jenkins-agent/Dockerfile_minimal

```diff
@@ -1,4 +1,4 @@
-ARG version=4.3-4-jdk11
+ARG version=4.3-8-jdk11
```

Comment:
can we use the .Dockerfile file extension please? syntax highlighting works then (at least in vscode)

i.e. minimal.Dockerfile and build.Dockerfile

Reply:
Sure. Good idea